zipkin-storage/cassandra/src/test/java/zipkin2/storage/cassandra/CassandraStorageExtension.java

```diff
@@ -31,57 +32,30 @@
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.output.Slf4jLogConsumer;
 import org.testcontainers.containers.wait.strategy.Wait;
-import org.testcontainers.utility.DockerImageName;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assumptions.assumeTrue;
+import static org.testcontainers.utility.DockerImageName.parse;
 import static zipkin2.Call.propagateIfFatal;
 import static zipkin2.storage.cassandra.ITCassandraStorage.SEARCH_TABLES;
 import static zipkin2.storage.cassandra.Schema.TABLE_DEPENDENCY;
 import static zipkin2.storage.cassandra.Schema.TABLE_SPAN;
 
 public class CassandraStorageExtension implements BeforeAllCallback, AfterAllCallback {
   static final Logger LOGGER = LoggerFactory.getLogger(CassandraStorageExtension.class);
-  static final int CASSANDRA_PORT = 9042;
-  final DockerImageName image;
-  GenericContainer<?> container;
-  CqlSession globalSession;
 
-  CassandraStorageExtension(DockerImageName image) {
-    this.image = image;
-  }
+  final CassandraContainer container = new CassandraContainer();
+  CqlSession globalSession;
 
   @Override public void beforeAll(ExtensionContext context) {
     if (context.getRequiredTestClass().getEnclosingClass() != null) {
       // Only run once in outermost scope.
       return;
     }
 
-    if (!"true".equals(System.getProperty("docker.skip"))) {
-      try {
-        LOGGER.info("Starting docker image " + image);
-        container = new GenericContainer<>(image)
-          .withExposedPorts(CASSANDRA_PORT)
-          .waitingFor(Wait.forHealthcheck());
-        container.start();
-        container.withLogConsumer(new Slf4jLogConsumer(LOGGER));
-      } catch (RuntimeException e) {
-        LOGGER.warn("Couldn't start docker image " + image + ": " + e.getMessage(), e);
-      }
-    } else {
-      LOGGER.info("Skipping startup of docker " + image);
-    }
-
-    try {
-      globalSession = tryToInitializeSession(contactPoint());
```

Comment:
most of the simplification here is dropping support of local connections (docker only now)